spec/lib/parse_spec.rb

```diff
@@ -486,6 +486,101 @@
     )
   end
 
+  it 'correctly finds expressions and function calls in a CREATE INDEX statment' do
```

Comment:
This could maybe be changed to `parses CREATE INDEX expressions` to follow the current naming convention

Reply:
The test right above it is `it "parses CREATE INDEX"`. I think I should probably just merge these--most DDL statements have a single test. I wasn't sure if this would be more elaborate and didn't want to complicate the simple test, but it's probably easier to do just one.

Reply:
I ended up merging these.